### PR TITLE
Adopt a code of conduct #301

### PR DESCRIPTION
**Pull request by inishchith**
PR for issue : #301 
Signed-off-by: Nishchith Shetty inishchith@gmail.com

### CODE REVIEW

**Comment by inishchith** [comment]
@pombredanne your review would be helpful

**Comment by pombredanne** [comment]
Thanks!
Let me review and think about the next step: we need community agreement on this :P

**Comment by inishchith** [comment]
@pombredanne Sure :P

**Comment by pombredanne** [comment]
Thank you! Looking good. Could you make this an .rst file instead of markdown?

**Comment by inishchith** [comment]
@pombredanne done 😄 , ping if any other changes required.

**Comment by pombredanne** [comment]
@SaravananOffl @sschuberth @haikoschol @majurg @mjherzog @jpopelka @JonoYang @jdaguil I need your feedback to see if you agree with the new code of conduct. I do 👍

**Comment by pombredanne** [comment]
I squashed and tag the commits BTW

**Comment by mjherzog** [comment]
The  code seems to need a specific email address for reporting issues. Not sure what that would be.

File: CODE_OF_CONDUCT.rst [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+Examples of unacceptable behavior by participants include:
+
+-  The use of sexualized language or imagery and unwelcome sexual
+   attention or advances
+-  Trolling, insulting/derogatory comments, and personal or political
+   attacks
+-  Public or private harassment
+-  Publishing others’ private information, such as a physical or
+   electronic address, without explicit permission
```
Comment (sschuberth) [inline CODE_OF_CONDUCT.rst]: Is accepting a PR which exposes the email address as part of the Git history *explicit* permission?

File: CODE_OF_CONDUCT.rst [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+official project e-mail address, posting via an official social media
+account, or acting as an appointed representative at an online or
+offline event. Representation of a project may be further defined and
+clarified by project maintainers.
+
+Enforcement
+-----------
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may
+be reported by contacting the project team at [INSERT EMAIL ADDRESS].
```
Comment (sschuberth) [inline CODE_OF_CONDUCT.rst]: This needs to be filled out.

File: CODE_OF_CONDUCT.rst [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+All complaints will be reviewed and investigated and will result in a
+response that is deemed necessary and appropriate to the circumstances.
```
Comment (sschuberth) [inline CODE_OF_CONDUCT.rst]: I would make clearer that *not* providing a response, e.g. in case of trolling, is also an option.

**Comment by majurg** [comment]
@pombredanne +1 from me, after @inishchith or others make the needed changes in the text

**Comment by SaravananOffl** [comment]
@pombredanne  +1 from me!

File: CODE_OF_CONDUCT.rst (hunk 1 above)
Comment (pombredanne) [reply CODE_OF_CONDUCT.rst]: Of course not... :) but you have a good point. I would be reluctant to change this standard CoC though after all this is only a template and we can make clear

**Comment by JonoYang** [comment]
@pombredanne :+1: after the changes

**Comment by pombredanne** [comment]
OK, so I will add a few things to fill in the blanks!

**Comment by pombredanne** [comment]
@inishchith How can I review and merge this with your fork deleted?

**Comment by inishchith** [comment]
@pombredanne really sorry for that 😞  . i had to open another ticket at #940 .  please have a look .

**Comment by pombredanne** [comment]
@inishchith that's OK, we are slow but eventually get down to every PR ;)

**Comment by pombredanne** [comment]
I will apply the comments there then :) thanks!

**Comment by pombredanne** [comment]
After some git mess, this is closed in favor of https://github.com/nexB/scancode-toolkit/pull/940

**Pull request closed without merging**